WINDOWS.md

```diff
@@ -0,0 +1,30 @@
+# Compiling and Installing **_libecl_** on Windows
+
+## Prerequisits:
+1. Download or clone the **_libecl_** repository to your local disk.
+
+2. Python 2.7 or 3.x installation
+   - Download a python instalation https://www.python.org/ or a python environment solution such as Anaconda https://anaconda.org/
+
+3. In command prompt, navigate to the **_libecl_** source directory you created in step 1. Use the python package manager **pip** to install **_libecl_** requirements via `pip install -r requirements.txt`
+    
+4. Download and install the community version of Microsoft Visual Studio https://visualstudio.microsoft.com/downloads/ . 
+Testing has been successful with Visual Studio 2017. At a minimum **_libecl_** requires the VS Studio packages for cmake, msbuild, c and c++ compilers (CL.exe).
+
+5. From the command prompt in step 3 one can activate the _Win64_ MSVC environment by running **vcvars64.bat**. This can be found in the MSVC install directory {PATH TO INSTALL}\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\
+
+6. Execute the build commands with the desired CMAKE parametres from `README.md`. The cmake generator should be Ninja.  
```

Comment:
I did not mean that we should push people to use `ninja` (well I'd say we should, but that's only my personal opinion).
An example is definitely good to have, and for that it may be worth using the generator for VS, as that's what most people will probably use. If anyone wants to play with cmake options, he/she probably can extrapolate enough info from that example

Reply:
Understood, I can put it back to VS generator but that example does not work for me. I cannot get it to acknowledge the `CMAKE_INSTALL_PREFIX` where as `ninja` works fine.